seeds/account_abstraction/erc4337/_schema.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+
+seeds:
```

Comment:
adding this seed schema file allows us to apply a `dunesql` tag and also apply dunesql data types